### ISSUE TITLE
V3: "required login to join" shows a unmeaning page

### ISSUE THREAD

**Issue by debuglevel** [bigbluebutton/greenlight#4198]
If I want to join a room with "required login to join = yes", this page appears:
<img width="843" alt="image" src="https://user-images.githubusercontent.com/2401265/203585283-b157e20f-c039-47fc-8c60-e65d0d0b4a97.png">

A roaster shows up, but disappears after a few seconds.

-> The page (not the roaster) should express what is going on und what I have to do.

**Comment by debuglevel** [comment]
Furthermore, the page redirects from `/rooms/eiu-w5m-bco-b9d` to `/`. I think that's a bad idea, because it just confuses people if they'd hit F5.

**Comment by farhatahmad** [comment]
Hmmm are you not getting this page? Are you sure you're going to `/rooms/id/join` and not just `/rooms/id`?
![image](https://user-images.githubusercontent.com/35435341/203734655-03fb0c8e-3e11-4b22-9a47-6c0c2a980b3f.png)

**Comment by debuglevel** [comment]
> Hmmm are you not getting this page? Are you sure you're going to `/rooms/id/join` and not just `/rooms/id`? ![image](https://user-images.githubusercontent.com/35435341/203734655-03fb0c8e-3e11-4b22-9a47-6c0c2a980b3f.png)

Good point! I might have fallen into this issue myself :) https://github.com/bigbluebutton/greenlight/issues/4200
